src/viam/app/provisioning_client.py

```diff
@@ -0,0 +1,95 @@
+from typing import Mapping, List, Optional
+
+from grpclib.client import Channel
+
+from viam import logging
+from viam.proto.provisioning import (
+    CloudConfig,
+    GetNetworkListRequest,
+    GetNetworkListResponse,
+    GetSmartMachineStatusRequest,
+    GetSmartMachineStatusResponse,
+    NetworkInfo,
+    ProvisioningServiceStub,
+    SetNetworkCredentialsRequest,
+    SetSmartMachineCredentialsRequest,
+)
+
+LOGGER = logging.getLogger(__name__)
+
+
+class ProvisioningClient:
+    """gRPC client for getting and setting smart machine info.
+
+    Constructor is used by `ViamClient` to instantiate relevant service stubs. Calls to
+    `ProvisioningClient` methods should be made through `ViamClient`.
+
+    Establish a connection::
+
+        import asyncio
+
+        from viam.rpc.dial import DialOptions, Credentials
+        from viam.app.viam_client import ViamClient
+
+
+        async def connect() -> ViamClient:
+            # Replace "<API-KEY>" (including brackets) with your API key and "<API-KEY-ID>" with your API key ID
+            dial_options = DialOptions.with_api_key("<API-KEY>", "<API-KEY-ID>")
+            return await ViamClient.create_from_dial_options(dial_options)
+
+
+        async def main():
+
+            # Make a ViamClient
+            viam_client = await connect()
+            # Instantiate a ProvisioningClient to run provisioning client API methods on
+            provisioning_client = viam_client.provisioning_client
```

Comment:
Don't you have to actually add the `provisioning_client` as a property on `viam_client` for this to work?

Reply:
🤦 yep you're totally right! Thanks for catching that.